public/js/util/validators.js

```diff
@@ -14,4 +14,30 @@ export const isHttpsUrl = (value) => {
     const error = new FieldError('not-https', 'Not a HTTPS url');
     return Promise.resolve(error);
   }
-};
\ No newline at end of file
+};
+
+export const checkItemsUnderWordCount = (values) => {
+    if (values.length > 0) {
+        var wordCount = values.map(function(itemData) {
+            var strippedText = stripHtml(itemData.body);
```

Comment:
Do you know where `itemData.body` is coming from? I'm thinking, if you can use the browser to do the stripping for you, that's even better (via `element.textContent`)

Reply:
This is the string value in the model, so unless we can first turn it into a DOM element I don't think so